.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+---
+
+**Describe the bug** A clear and concise description of what the bug is.
```

Comment:
Maybe use `h4` for headings with `####`? The descriptions could go on their own lines, and be wrapped in comments so they don't appear if people leave them in:

```md
#### Describe the bug

<!-- A clear and concise description of what the bug is -->
```